assets/src/edit-story/components/checklist/checks/storyAmpValidationErrors.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { trackEvent } from '@web-stories-wp/tracking';
+import { useEffect, useRef, useState } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+import { ChecklistCard, DefaultFooterText } from '../../checklistCard';
+import { PRIORITY_COPY } from '../constants';
+import { useRegisterCheck } from '../countContext';
+
+export async function getStoryAmpValidationErrors({ link, status }) {
+  // TODO: Enable for drafts as well?
```

Comment:
I am not familiar enough to know if we should enable this for drafts as well. To me, seems like something that could go either way.

Reply:
I think enabling for drafts is probably less confusing. Do we have any other PPC items that don't run on drafts?

Reply:
nope! everything runs on drafts otherwise. i thought maybe this rule here was mentioned because the validation check doesn't work well on a draft since those aren't done? 

we can swap it. have anything with a link run.

Reply:
(see my comment on the below thread, i think this should stay at just after publish. Most if not all issues I've been able to get in the editor resolve on save or publish.)